parts/windows/windowscsehelper.ps1

```diff
@@ -217,3 +217,16 @@ function Assert-FileExists {
         Set-ExitCode -ExitCode $global:WINDOWS_CSE_ERROR_FILE_NOT_EXIST -ErrorMessage "$Filename does not exist"
     }
 }
+
+function Get-WindowsVersion {
+    param (
+        [Parameter(Mandatory = $true)][string]
+        $buildNumber

```

Comment:
Can we not use a parameter `$buildNumber` but get it inside this function?

Reply:
Good idea. Thanks for guidance.